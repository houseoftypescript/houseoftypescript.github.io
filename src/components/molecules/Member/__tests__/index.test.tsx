import { render } from '@testing-library/react';
import { StaticImageData } from 'next/image';
import { Member } from '../index';

describe('Member', () => {
  it('renders correctly', () => {
    const button = render(<Member avatar={{} as StaticImageData}></Member>);
    expect(button.container).toMatchSnapshot();
  });
});
