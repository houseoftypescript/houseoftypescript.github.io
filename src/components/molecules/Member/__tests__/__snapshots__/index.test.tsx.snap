// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Member renders correctly 1`] = `
<div>
  <div>
    <img
      alt=""
      class="rounded-xl border"
      data-nimg="1"
      decoding="async"
      loading="lazy"
      src=""
      style="color: transparent;"
    />
    <h2
      class="mt-4 text-lg md:text-2xl font-medium"
    />
    <p
      class="mt-2 text-sm md:text-lg text-gray-500"
    />
    <div
      class="mt-2 flex gap-2"
    />
  </div>
</div>
`;
